### PR TITLE
chore: update to Dash Core v0.15.0

### PR DESCRIPTION
**Pull request by thephez**
Use 0.15.0.0 for image

### CODE REVIEW

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by nmarley — APPROVED** [review]
utACK

**Comment by thephez** [comment]
I do not have write access here, so someone else will have to merge.

**Comment by nmarley** [comment]
@thephez Just built and waiting for the push, will merge shortly...

**Pull request merged**
